paasta_tools/paasta_cli/utils.py

```diff
@@ -540,3 +540,45 @@ def get_jenkins_build_output_url():
     if build_output:
         build_output = build_output + 'console'
     return build_output
+
+
+class PaastaValidateMessages:
+
+    """Collection of messages printed out by 'paasta validate'.
+    Helpful as it avoids cumbersome maintenance of the unit tests.
+    """
+
+    SCHEMA_VALID = success("Successfully validated schema")
+    SCHEMA_INVALID = failure(
+        "Failed to validate schema. More info:", "http://y/yelpsoa-configs")
+
+
+class PaastaValidateSchemas:
```

Comment:
again, these schemas are very specific to the validate module, so shouldn't be in utils.py + don't really belong in a class, either.